code/datums/wires/_wires.dm

```diff
@@ -173,6 +180,14 @@
 /datum/wires/proc/pulse(wire, user)
 	if(is_cut(wire))
 		return
+
+	if(usr.mind?.cm_skills && usr.mind.cm_skills.engineer < SKILL_ENGINEER_ENGI)
```

Comment:
this should be above is_cut check since wires could be cut while someone is fumbling

Reply:
That was done on purpose to reduce the frustration for the users - though I forgot the is_cut check in the do-after.